pkg/tracker/disputeChecker.go

```diff
@@ -43,10 +46,9 @@ type ValueCheckResult struct {
 }
 
 // CheckValueAtTime queries for the details regarding the disputed value.
-func CheckValueAtTime(reqID uint64, val *big.Int, at time.Time) *ValueCheckResult {
-	cfg := config.GetConfig()
-	//
+func CheckValueAtTime(config *config.Config, reqID uint64, val *big.Int, at time.Time) *ValueCheckResult {
```

Comment:
```suggestion
func CheckValueAtTime(cfg *config.Config, reqID uint64, val *big.Int, at time.Time) *ValueCheckResult {
```
I think having the var name the same at the package confuses the compiler.